### PR TITLE
Refactor shooter

### PR DESCRIPTION
**Pull request by ShayanHasNo**

### CODE REVIEW

**Review by Aradig26 — CHANGES_REQUESTED** [review]
rename branch and commit

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -31,7 +31,9 @@
 import frc.robot.subsystems.indexer.Indexer;
 import frc.robot.subsystems.indexer.IndexerIO;
 import frc.robot.subsystems.intake.Intake;
+import frc.robot.subsystems.intake.IntakeConstants;
 import frc.robot.subsystems.intake.IntakeIO;
+import frc.robot.subsystems.intake.IntakeIOPhysical;
```
Comment (EQMoore) [inline src/main/java/frc/robot/RobotContainer.java:36]: remove these imports, besides line 35 (IntakeIO), after you remove all requested references of intake in this file

**Review by EQMoore — CHANGES_REQUESTED** [review]
It looks like you have intake changes in this pull request, please remove them as requested.

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -172,7 +177,9 @@ public RobotContainer() {
    */
   private void configureButtonBindings() {
     driverController.y().onTrue(Commands.runOnce(() -> isRobotOriented = !isRobotOriented));
-
+    driverController.x().whileTrue(intake.intake(IntakeConstants.INTAKE_SPEED));
+    driverController.b().whileTrue(intake.release(IntakeConstants.RELEASE_SPEED));
+    driverController.leftTrigger().whileTrue(intake.intake(driverController.getLeftTriggerAxis()));
```
[the remark this answers is not in the document]
Comment (EQMoore) [reply src/main/java/frc/robot/RobotContainer.java:182]: Also, why is intake in a shooter pull request?

**Review by EQMoore — APPROVED** [review]
lgtm

**Review by Aradig26 — APPROVED** [review]

**Pull request merged**